sources/engine/Xenko.Physics/Shapes/HeightfieldColliderShape.cs

```diff
@@ -48,7 +68,7 @@ bool flipQuadEdges
             HeightStickWidth = heightStickWidth;
             HeightStickLength = heightStickLength;
             HeightType = heightType;
-            HeightScale = heightScale;
+            HeightScale = (HeightType == HeightfieldTypes.Float) ? 1f : heightScale;
```

Comment:
Seems strange/misleading to ignore this parameter only in case of float?

Reply:
I revert that.
That is unnecessary because HeightfieldColliderShapeDesc does it.